.github/workflows/test-dev.yml

```
@@ -20,14 +20,102 @@ permissions:
 
 env:
+  VERSION: 0.7.96
   CONTAINER_REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}-devcontainer
   TOTAL_TIMEOUT: 900
   INSTALL_TIMEOUT: 300
 
 jobs:
+  devcontainer:
+    name: "Build: Dev Container"
+    id: devcontainer
```

actionlint (1.7.4)

unexpected key "id" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)